### PR TITLE
legacy: remove deprecated code (bug 1870142)

### PR DESCRIPTION
**Pull request by zzzeid**
- remove files that are no longer used

### CODE REVIEW

File: src/lando/api/legacy/cache.py
Comment (zzzeid) [inline src/lando/api/legacy/cache.py:1]: Uses django's cache functionality now.

File: src/lando/api/legacy/cli.py
Comment (zzzeid) [inline src/lando/api/legacy/cli.py:1]: All commands have been ported or are deprecated, except for the pre/post deploy ones. Those are tracked in [bug 1914267](https://bugzilla.mozilla.org/show_bug.cgi?id=1914267).

File: src/lando/api/legacy/dockerflow.py
Comment (zzzeid) [inline src/lando/api/legacy/dockerflow.py:1]: Already ported into the dockerflow app.

File: src/lando/api/legacy/redis.py
Comment (zzzeid) [inline src/lando/api/legacy/redis.py:1]: No longer needed.

File: src/lando/api/legacy/ui.py
Comment (zzzeid) [inline src/lando/api/legacy/ui.py:1]: No longer needed.

File: src/lando/api/legacy/wsgi.py
Comment (zzzeid) [inline src/lando/api/legacy/wsgi.py:1]: No longer needed.

File: src/lando/api/tests/test_redis.py
Comment (zzzeid) [inline src/lando/api/tests/test_redis.py:1]: No longer needed.

File: src/lando/ui/legacy/helpers.py
Comment (zzzeid) [inline src/lando/ui/legacy/helpers.py:1]: Already ported, no longer needed, except for what is tracked in [bug 1912972](https://bugzilla.mozilla.org/show_bug.cgi?id=1912972).

File: src/lando/ui/legacy/landoapi.py
Comment (zzzeid) [inline src/lando/ui/legacy/landoapi.py:1]: No longer needed.

File: src/lando/api/legacy/app.py
Comment (zzzeid) [inline src/lando/api/legacy/app.py:1]: No longer needed.

**Review by dklawren — APPROVED** [review]

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**